src/SurfaceFluxes.jl

```diff
@@ -459,10 +496,10 @@ function compute_bstar(param_set, L_MO, sc::AbstractSurfaceConditions{FT}, uft::
     grav::FT = SFP.grav(param_set)
     DSEᵥ_sfc = TD.virtual_dry_static_energy(thermo_params, ts_sfc(sc), grav * z_sfc(sc))
     DSEᵥ_in = TD.virtual_dry_static_energy(thermo_params, ts_in(sc), grav * z_in(sc))
+    ΔDSEᵥ = DSEᵥ_in - DSEᵥ_sfc
     DSEᵥ_star =
         compute_physical_scale_coeff(param_set, sc, L_MO, UF.HeatTransport(), uft, scheme) * (DSEᵥ_in - DSEᵥ_sfc)
-
-    return -grav * DSEᵥ_star / DSEᵥ_in
+    return grav * DSEᵥ_star / DSEᵥ_in
```

Comment:
good catch, my mistake. we should update the design doc as well

Reply:
Yes, I did a quick update to the design doc, but it would be good if you go over it again whenever you have time